src/pages/Report.tsx

```diff
@@ -0,0 +1,212 @@
+                    <li>busca e tenha um histórico de aprendizado contínuo;</li>
+                    <li>
+                      busca e fomenta a colaboração, mas também tenha as suas
+                      contribuições individuais inovadoras;
+                    </li>
+                    <li>
+                      entenda o objetivo e propósito da empresa, e consiga atuar
+                      com autonomia dentro de suas responsabilidades.
+                    </li>
+                  </ul>
+                  <p className="font-medium text-gray-600 my-3">
+                    {" "}
+                    Responsabilidades e atribuições:
+                  </p>
+                  <p className="font-medium text-gray-600 my-3">
+                    Culturalmente, procuramos uma pessoa que:
+                  </p>
+                  <ul className="list-disc ml-5 text-gray-600 text-sm">
+                    <li>seja curiosa por natureza;</li>
+                    <li>perfil mão na massa e ágil;</li>
+                    <li>sabe ouvir e se comunicar;</li>
+                    <li>busca e tenha um histórico de aprendizado contínuo;</li>
+                    <li>
+                      busca e fomenta a colaboração, mas também tenha as suas
+                      contribuições individuais inovadoras;
+                    </li>
+                    <li>
+                      entenda o objetivo e propósito da empresa, e consiga atuar
+                      com autonomia dentro de suas responsabilidades.
+                    </li>
+                  </ul>
+                  <p className="font-medium text-gray-600 my-3">
+                    {" "}
+                    Responsabilidades e atribuições:
+                  </p>
+                  <p className="font-medium text-gray-600 my-3">
+                    Culturalmente, procuramos uma pessoa que:
+                  </p>
+                  <ul className="list-disc ml-5 text-gray-600 text-sm">
+                    <li>seja curiosa por natureza;</li>
+                    <li>perfil mão na massa e ágil;</li>
+                    <li>sabe ouvir e se comunicar;</li>
+                    <li>busca e tenha um histórico de aprendizado contínuo;</li>
+                    <li>
+                      busca e fomenta a colaboração, mas também tenha as suas
+                      contribuições individuais inovadoras;
+                    </li>
+                    <li>
+                      entenda o objetivo e propósito da empresa, e consiga atuar
+                      com autonomia dentro de suas responsabilidades.
+                    </li>
+                  </ul>
+                  <p className="font-medium text-gray-600 my-3">
+                    {" "}
+                    Responsabilidades e atribuições:
+                  </p>
+                </div>
+              </div>
+              <div className="bg-[#FCFCFC] border rounded-lg mt-6 py-4 px-6 flex items-center justify-between hidden md:flex">
```

Comment:
Remover classes iguais para `md `e < md
```suggestion
              <div className="bg-[#FCFCFC] border rounded-lg mt-6 py-4 px-6 flex items-center justify-between hidden">
```

Reply:
done